<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimerAttrCfger</class>
 <widget class="QGroupBox" name="TimerAttrCfger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>230</height>
   </rect>
  </property>
  <property name="title">
   <string>Timer</string>
  </property>
  <property name="alignment">
   <set>Qt::AlignCenter</set>
  </property>
  <property name="flat">
   <bool>true</bool>
  </property>
  <widget class="QRadioButton" name="countDown">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Count Down</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="elapse">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Elapse</string>
   </property>
  </widget>
  <widget class="QLabel" name="targetTag">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>61</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Target:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="target">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="extraText">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>251</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Extra Text</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QLineEdit" name="text">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>231</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorComboBox" name="textColor">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>41</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Color:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="extraLine">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Extra Line</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="underLine">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>30</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Underline</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>image/icon/text_underline_64.png</normaloff>image/icon/text_underline_64.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="point">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>30</y>
     <width>41</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Font size</string>
   </property>
   <property name="maximum">
    <number>512</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QPushButton" name="italic">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>30</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Italic</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>image/icon/text_italic_64.png</normaloff>image/icon/text_italic_64.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFontComboBox" name="family">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="fontFilters">
    <set>QFontComboBox::MonospacedFonts</set>
   </property>
  </widget>
  <widget class="QPushButton" name="bold">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>30</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Bold</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>image/icon/text_bold_64.png</normaloff>image/icon/text_bold_64.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="ColorComboBox" name="digitColor">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="minute">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>61</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Minute</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Digit Color:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="day">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>51</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Day</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="hour">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>51</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Hour</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="unit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Unit</string>
   </property>
  </widget>
  <widget class="ColorComboBox" name="unitColor">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="second">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>61</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Second</string>
   </property>
  </widget>
  <widget class="QLabel" name="unitColorTag">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>41</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Color:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorComboBox</class>
   <extends>QComboBox</extends>
   <header>ColorComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
